Source/Icebreaker/Helpers/UserInfo.cs

```diff
@@ -39,10 +39,10 @@ public string UserId
         public string ServiceUrl { get; set; }
 
         /// <summary>
-        /// Gets or sets a value indicating whether the user is opted in to pairups
+        /// Gets or sets a value indicating whether the user is opted in to pairups for each team the user is in
         /// </summary>
         [JsonProperty("optedIn")]
-        public bool OptedIn { get; set; }
+        public Dictionary<string, bool> OptedIn { get; set; }
```

Comment:
I thought that a dictionary would be the most straightforward way to keep track of all teams as well as opt in status for each team so that pulling data for view teams is easier.